conf/gis.config

```diff
@@ -0,0 +1,27 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for GIS (Aquila)
+ * -------------------------------------------------
+ * Defines reference genomes, using iGenome paths
+ * Imported under the default 'standard' Nextflow
+ * profile in nextflow.config
+ */
+
+process {
+  executor = 'sge'
+  clusterOptions = { "-l mem_free=" + task.memory.toString().replaceAll(/[\sB]/,'') }
+  penv = 'OpenMP'
+  maxRetries = 3
+  errorStrategy = { task.attempt < 2 ? 'retry' : 'finish' }
+  // auto translate container name into conda environment name
+  beforeScript = { 'source /mnt/projects/rpd/rc/init.2017-04; module load miniconda3; set +u; source activate nfcore-rnaseq-1.5dev; set -u;' }
```

Comment:
We'll hopefully have native support for conda very soon: https://github.com/nf-core/cookiecutter/issues/27

Reply:
Great. Thanks. Looking forward.